alonzo/formal-spec/protocol-parameters.tex

```diff
@@ -6,29 +6,40 @@ \section{Language Versions and Cost Models}
 
 \vspace{12pt}
 \begin{tabular}{lp{5in}}
+  $\CostMod$ &
+  A term of this type represents the vector of coefficients that are used to generate
```

Comment:
I've already mentioned on the other PR that 'term of this type' isn't great, because it actually means something different. I think 'element' or 'member' is just fine here and at the end of the sentence.